frontend/app/controllers/Outages.scala

```diff
@@ -3,7 +3,7 @@ package controllers
 import play.api.mvc.Controller
 
 object Outages extends Controller {
-  def maintenanceMessage = CachedAction {
+  def maintenanceMessage = NoCacheAction {
```

Comment:
actually why is this changing to no cache?  the actual url is a [temporary redirect](https://github.com/guardian/membership-frontend/blob/395d9fdcbe54cdf670edac075ac0a06a833e0570/frontend/app/actions/Fallbacks.scala#L14) and it's that that shouldn't be cached.